ooniprobe/AppDelegate.mm

```diff
@@ -114,8 +121,13 @@ - (void)alertView:(UIAlertView *)alertView clickedButtonAtIndex:(NSInteger)butto
     if (buttonIndex == 1 && alertView.tag == 1){
         [self openBrowser];
     }
+    else if (buttonIndex == 1 && alertView.tag == 2){
```

Comment:
@lorenzoPrimi it would probably help to have `#define`s for these numbers, or it's a bit tricky to follow the code.